Makefile

```diff
@@ -126,14 +126,14 @@ vendor: $(DOCKER_PHP_CONTAINER_FLAG) composer.json $(wildcard composer.lock) $(w
 	@touch -c $@ composer.json .env composer.lock
 
 .PHONY: docker-start-if-not-running
-docker-start-if-not-running: ### some xxx
```

Comment:
oups :innocent: 

BTW I remove the doc because I don't think this target will be directly used by users...